services/src/main/scala/cromwell/services/metadata/WorkflowQueryKey.scala

```diff
@@ -168,17 +175,35 @@ sealed trait SeqWorkflowQueryKey[A] extends WorkflowQueryKey[Seq[A]] {
 
 sealed trait IntWorkflowQueryKey extends WorkflowQueryKey[Option[Int]] {
   override def validate(grouped: Map[String, Seq[(String, String)]]): ErrorOr[Option[Int]] = {
-    valuesFromMap(grouped) match {
+    valuesFromMap(grouped).toList match {
       case vs if vs.lengthCompare(1) > 0 =>
         s"Found ${vs.size} values for key '$name' but at most one is allowed.".invalidNel[Option[Int]]
       case Nil => None.validNel
       case v :: Nil =>
         Try(v.toInt) match {
           case Success(intVal) => if (intVal > 0) Option(intVal).validNel else s"Integer value not greater than 0".invalidNel[Option[Int]]
-          case _ => s"Value given for $displayName does not parse as a integer: $v".invalidNel[Option[Int]]
+          case _ => s"Value given  for $displayName does not parse as a integer: $v".invalidNel[Option[Int]]
```

Comment:
🔙

Reply:
This one is 'oops!'. Thanks for pointing it out :)